src/config/chis-tg/config.json

```diff
@@ -10,8 +10,8 @@
       "domain": "moh-togo-uat.dev.medicmobile.org"
     },
     {
-      "friendly": "MoH Togo PROD",
-      "domain": "moh-togo.app.medicmobile.org"
+      "friendly": "Local Dev Togo",
+      "domain": "127-0-0-1.local-ip.medicmobile.org:10443"
```

Comment:
please revert this and [use development mode](https://github.com/medic/cht-user-management/?tab=readme-ov-file#development):

```
NODE_ENV===dev
```

and then set the `CHT_DEV_URL_PORT` and `CHT_DEV_HTTP` env vars

Reply:
Looking at [the latest commit in this branch](https://github.com/medic/cht-user-management/blob/32a934137f086ffc3107af7a17f5d108385d8ad4/src/config/chis-tg/config.json#L14), I see that Production is still missing and the local dev instance is there.  Please revert, we want to just use `NODE_ENV` as `dev` and keep production URL .

Reply:
we still need to revert this change - as is this removes the option for production and replaces it with a dev instance.